README.md

```diff
@@ -6,8 +6,21 @@ Zinc is a stand-alone version of [sbt]'s incremental compiler.
 Download the [latest stable version][download].
 
 [sbt]: http://github.com/sbt/sbt
-[download]: http://downloads.typesafe.com/zinc/0.3.5.3/zinc-0.3.5.3.tgz
+[download]: http://downloads.typesafe.com/zinc/0.3.7/zinc-0.3.7.tgz
 
+Status
+------
+
+Zinc is currently in a maintenance mode. There's no development of new features and new
+releases are meant to stay up-to-date with latest sbt releases. Zinc tries to follow
+final releases of sbt. It doesn't follow milestone releases but exceptions can be
+introduced on a case-by-case basis.
+
+Maintenance of zinc is performed on best-effort terms. We try our best to not lag
+behind sbt with zinc releases but we do not guarantee it. Occasional delays can
+happen due to other priorities.
+
+If you are interested in helping with zinc maintenance or development, please contact [Grzegorz Kossakowski](grek@typesafe.com) from Typesafe.
```

Comment:
Let's update the README once those plans are made public. Check my suggestion on publishing the design doc as an issue in sbt's issue tracker.

Also, I'd argue that this new thing should be just called zinc :-)